src/navigation/Navigator.tsx

```diff
@@ -34,7 +35,8 @@ const styles = StyleSheet.create({
     shadowOpacity: 0,
     elevation: 0,
     borderBottomColor: COLORS.disabled,
-    borderBottomWidth: 1
+    borderBottomWidth: 1,
+    height: wp('15%')
```

Comment:
height depending on the width. Wouldn't `hp(..)` make more sense here?

Reply:
the problem is that font, spacings and everything depend on the wp, so for tablets its increasing a lot and doesn't fit with hp for tablet and mobile

![image](https://user-images.githubusercontent.com/37902063/154801129-117c6967-3491-418b-83a9-ed7a38d29f80.png)